packages/react-server-cli/src/callerDependency.js

```diff
@@ -0,0 +1,7 @@
+import path from "path";
+
+export default function callerDependency(dep) {
+	// TODO: We should grab stuff based on what the routes file would get out
+	// of `require.resolve(dep)`.  Using `process.cwd()` instead for now.
+	return path.resolve(path.join(process.cwd(), "node_modules/" + dep));
```

Comment:
Maybe use https://www.npmjs.com/package/look-up to look up the file system until you find `.reactserverrc` and treat that as the root of the project?